src/hooks/Mobile.ts

```diff
@@ -1,3 +1,4 @@
+import debounce from 'lodash.debounce'
```

Comment:
**question**: is it possible to use `lodash/debounce` instead? This way we can avoid an extra dependency.

Reply:
makes sense!